dev/docker_image/Dockerfile

```diff
@@ -108,6 +108,9 @@ ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/software/kentUtils_bin_v377/lib
 # Instal Trimmomatic JAR
 RUN wget http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.39.zip && unzip Trimmomatic-0.39.zip && mv Trimmomatic-0.39/trimmomatic-0.39.jar trimmomatic.jar && chmod +rx trimmomatic.jar && rm -rf Trimmomatic-0.39.zip Trimmomatic-0.39/
 
+# Install pytest for testing environment
+RUN pip3 install --no-cache-dir pytest
```

Comment:
See related PR on ATAC, it would be better to keep dev dependencies outside of the pipeline docker image.

Reply:
Yes, I agree. I will plan to remove it in the next code refactoring probably in the near future.